### PR TITLE
change http to https

### PR DESCRIPTION
**Pull request by knoxcard2**
change http to https

### CODE REVIEW

File: lib/sitemap-stream.ts [hunk 1]
```diff
@@ -1,79 +1,21 @@
...
-    callback();
-  }
-
-  _flush(cb: TransformCallback): void {
-    this.push(closetag);
-    cb();
-  }
+import { Transform, TransformOptions, TransformCallback, Readable } from 'stream';
+import { ISitemapOptions } from './sitemap';
+export declare const preamble = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><urlset xmlns=\"https://www.sitemaps.org/schemas/sitemap/0.9\" xmlns:news=\"https://www.google.com/schemas/sitemap-news/0.9\" xmlns:xhtml=\"https://www.w3.org/1999/xhtml\" xmlns:image=\"https://www.google.com/schemas/sitemap-image/1.1\" xmlns:video=\"https://www.google.com/schemas/sitemap-video/1.1\">";
```
Comment (derduher) [inline lib/sitemap-stream.ts]: xmlns is not the same as a URL you would type into your browser. Changing this to https would produce an invalid document. xmlns must match the namespace declared in the xsd.

File: lib/sitemap-stream.ts [hunk 2]
```diff
@@ -1,79 +1,21 @@
...
-              drain = Buffer.concat([drain, chunk]);
-            }
-            next();
-          },
-        })
-      )
-      .on('error', reject)
-      .on('finish', () => resolve(drain));
-  });
-}
```
Comment (derduher) [inline lib/sitemap-stream.ts]: why did you delete a whole bunch of code?

File: lib/sitemap-stream.ts (hunk 1 above)
Comment (knoxcard2) [reply lib/sitemap-stream.ts]: Crud! I hate http so much, my insecurities get to me.  Just nuked them, submitted a blind pull request and here we are.  My apologies.

File: lib/sitemap-stream.ts (hunk 2 above)
Comment (knoxcard2) [reply lib/sitemap-stream.ts]: lol, i am going to blame my keyboard.

**Comment by knoxcard2** [comment]
My sincerest apologies @derduher, this pull request is muck!  I will close it now.

**Comment by derduher** [comment]
no worries!
